.expeditor/scripts/build_mac_hab_binary.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/local/bin/bash
+
+set -euo pipefail
+
+source .expeditor/scripts/shared_release_habitat.sh
+
+# Get secrets! (our auth token and aws creds should be auto-injected but there's a bug:
+# https://github.com/chef/ci-studio-common/issues/200)
+eval "$(vault-util fetch-secret-env)"
+
+export HAB_AUTH_TOKEN="${ACCEPTANCE_HAB_AUTH_TOKEN}"
+export HAB_BLDR_URL="${ACCEPTANCE_HAB_BLDR_URL}"
+
+channel=$(get_release_channel)
+
+echo "--- Channel: $channel - bldr url: $HAB_BLDR_URL"
+
+echo "--- Installing buildkite agent"
+brew tap buildkite/buildkite
+brew install --token="$BUILDKITE_AGENT_ACCESS_TOKEN" buildkite-agent
+
+echo "--- Installing mac bootstrap package"
+# TODO: subscribe to releases: https://github.com/habitat-sh/release-engineering/issues/84
+bootstrap_package_version="$(cat MAC_BOOTSTRAPPER_VERSION)"
+bootstrap_package_name="mac-bootstrapper-${bootstrap_package_version}-1"
+curl "https://packages.chef.io/files/current/mac-bootstrapper/${bootstrap_package_version}/mac_os_x/10.12/${bootstrap_package_name}.dmg" -o "${bootstrap_package_name}.dmg"
```

Comment:
Seem like we could eliminate the 
```suggestion
curl "https://packages.chef.io/files/current/mac-bootstrapper/${bootstrap_package_version}/mac_os_x/10.12/${bootstrap_package_name}.dmg" -O"
```